aws_s3_bucket.tf

```diff
@@ -29,13 +33,17 @@ resource "aws_s3_bucket" "s3BucketSseRulesWithKmsNull" {
       }
     }
   }
+
+  versioning {
+    enabled = true
+  }
 }
 
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-